com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -648,6 +648,7 @@ public void NetworkUpdate(NetworkUpdateStage updateStage)
 
         private void OnNetworkEarlyUpdate()
         {
+            NotifyProfilerListeners();
```

Comment:
coming from: #641 — fix: fixed network activity occurring outside the core update getting missed in the profiler